### DISCUSSION TITLE
Toggle cooldowns acording to target

### DISCUSSION THREAD

**Discussion by johjoh3105** [Hekili/hekili#762] [Ideas]
hey. I'm really liking this addon, and as an Altoholic it's realy helpfull for me.
BUT. there's one thing that anoys me. I'm getting the major CD's show up when I'm fighting super easy enemies.
is there a way to make CD's show only if current target is a "tough enemy" such as a rare/elite? or could such an option be made without too much coding issues?

thanks for your time,

**Comment by Hekili** [comment]
Probably not.  The biggest reason is that Blizzard is really inconsistent about identifying rareness/difficulty of enemies, so it would be wrong often enough that people would probably find it annoying.

The best solution is to use your CD toggle keybind and control this manually.  It's pretty important to get used to doing that, as you'll find yourself turning CDs on/off in raid encounters as well.

**Reply by johjoh3105** [reply]
Yeah, after poking around in the options I found a setting that seperate the cd's and interupts from the main recomendation bar, and that was much better :3
